.circleci/run_osx_build.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -x
+
+echo "Removing homebrew from Circle CI to avoid conflicts."
+curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall > ~/uninstall_homebrew
+chmod +x ~/uninstall_homebrew
+~/uninstall_homebrew -fq
+rm ~/uninstall_homebrew
+
+echo "Installing a fresh version of Miniconda."
+MINICONDA_URL="https://repo.continuum.io/miniconda"
+MINICONDA_FILE="Miniconda3-latest-MacOSX-x86_64.sh"
+curl -L -O "${MINICONDA_URL}/${MINICONDA_FILE}"
+bash $MINICONDA_FILE -b
+
+echo "Downloading sdk"
+curl -L -O https://github.com/phracker/MacOSX-SDKs/releases/download/10.13/MacOSX10.9.sdk.tar.xz
+tar -xf MacOSX10.9.sdk.tar.xz -C $(xcode-select -p)/Platforms/MacOSX.platform/Developer/SDKs/
+sed -i '' "s|<string>10.11</string>|<string>10.9</string>|g" $(xcode-select -p)/Platforms/MacOSX.platform/Info.plist
```

Comment:
I'm not a Mac user but I'll check the logs and see if I can find anything useful to you.

Reply:
I think we want an actual mac user with 10.9 to see if the conda package produced by circle-ci works. cc @jakirkham 

Btw, we can ssh to these Mac workers, so that's a huge help for me.

Reply:
> Btw, we can ssh to these Mac workers, so that's a huge help for me.

Indeed that is very helpful.

Reply:
Thanks guys! I only have access to OSX 10.11 I'm afraid so not very useful for testing...